root/report/recordings_same_name_different_artists_same_name.tt

```diff
@@ -0,0 +1,19 @@
+[%- WRAPPER 'layout.tt' title=l('Recordings with the same name by different artists with the same name') full_width=1 -%]
+
+<h1>[% l('Recordings with the same name by different artists with the same name') %]</h1>
+
+<ul>
+    <li>[% l('This report shows all recordings with the same name that have different artist MBIDs with the same name.') %]</li>
+    <li>[% l('Currently, this only works with recordings that have one artist.') %]</li>
```

Comment:
Nitpick: I would add slightly more context to this string by doing s/this/this report/. Also, move this below the `These are most likely ...` item because it's less important than that.